applications/FROST.md

```diff
@@ -0,0 +1,265 @@
+- Has graduated the PBA Hong Kong cohort. 
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 2 month
+- **Full-Time Equivalent (FTE):**  1 FTE
+- **Total Costs:** 10,000 USD
+
+### Milestone 1 - FROST Protocol Libraries Implementation
+
+- **Estimated duration:** 1 month
+- **FTE:** 1
+- **Costs:** 5,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | GPLv3 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can use both libraries to implement the FROST protocol, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Article | We will publish an **article**/workshop that explains how the FROST protocol and the libraries work. |
+| **1.a** | Client-side library | We will implement a client-side library for the FROST protocol compatible with Substrate in Typescript. |
+| **1.b** | Server-side library | We will implement a server-side library for the FROST protocol compatible with Substrate in Rust. |
+
+### Milestone 2 - Proof of Concept Threshold Signature Implementation
+
+- **Estimated duration:** 1 month
+- **FTE:** 1
+- **Costs:** 5,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | GPLv3 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a group of users can generate a shared public key and use the corresponding secret key to sign threshold signatures multiple times without revealing it, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Article | We will publish an **article**/workshop that explains how the Proof of Concept works under the hood and how it can be used from a user perspective. |
+| **1.a** | Client Implementation | We will implement a client in Typescript compatible with Substrate as a fork of Talisman (https://github.com/TalismanSociety/talisman) using the client-side library. |
+| **1.b** | Server Implementation | We will implement a server in Rust compatible with Substrate using the server-side library. |
+| **1.c** | Proof of Concept Threshold Signature Implementation | We will integrate both the client and the server as a Proof of Concept Threshold Signature implementation compatible with Substrate and ready to be utilized by the end user. |
```

Comment:
Could you add the Docker deliverable here as well, as it's usually [mandatory](https://github.com/w3f/Grants-Program/blob/e7b653afa42fff1f91c7d4ba9a69c2bf07edf63d/applications/application-template.md#milestone-1-example--basic-functionality)?

Reply:
I force pushed by mistake doing this, sorry.